packages/app/src/app/components/Preview/DevTools/React-Devtools/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { initialize } from 'react-devtools-inline/frontend';
+
+import { Container } from './elements';
+
+const DevTools = () => {
+  const ReactDevTools = useRef(null);
+  const [devToolsReady, setDtReady] = useState(false);
+
+  useEffect(() => {
+    const iframe = document.getElementById(
+      'sandbox-preview'
+    ) as HTMLIFrameElement;
+    const contentWindow = iframe.contentWindow;
+    ReactDevTools.current = initialize(contentWindow);
+    iframe.onload = () => {
+      contentWindow.postMessage(
+        {
+          type: 'activate',
+        },
+        '*'
+      );
+    };
+    setDtReady(true);
+  }, []);
+
+  return (
+    <Container>
+      {devToolsReady && ReactDevTools.current && <ReactDevTools.current />}
+    </Container>
+  );
+};
+
+export default {
+  id: 'codesandbox.devtools',
+  title: 'React DevTools',
```

Comment:
Btw instead of "React DevTools" I wonder if we should show "⚛️ Components" (to better align with the browser extension)? I wonder if that would be more or less confusing? :smile:

Reply:
Naa, that makes sense!

Done :)